src/main/scala/codacy/detekt/Detekt.scala

```diff
@@ -84,7 +84,7 @@ object Detekt extends Tool {
           val patterns = patternsRaw.view.map {
             case (patternId, patternDef) =>
               val parameters = patternDef
-                .flatMap(_.parameters)
+                .flatMap(pattern => Some(pattern.parameters))
```

Comment:
Can't this be replaced with a map and remove the Some?
```suggestion
                .map(_.parameters)
```
Disregard if this is incorrect